tests/libtest/test_ibmcloud.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf8 -*-
+
+
+import logging
+
+from ocs_ci.ocs.platform_nodes import IBMCloud
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import libtest
+from ocs_ci.ocs import constants, node
+from ocs_ci.ocs.node import get_all_nodes
+from ocs_ci.framework.pytest_customization.marks import ibmcloud_platform_required
+
+from ocs_ci.utility.utils import run_cmd
+
+logger = logging.getLogger(__name__)
+
+
+@libtest
+@ibmcloud_platform_required
+def test_rebootnodes():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+    worker_nodes = node.get_nodes(node_type="worker", num_of_nodes=1)
+    ibmcloud.restart_nodes(worker_nodes)
+
+@libtest
+@ibmcloud_platform_required
+def test_attachvolume():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+
+    volume = ibmcloud.getVolumeId()
+
+    worker_nodes = node.get_nodes(node_type="worker", num_of_nodes=1)
+    ibmcloud.attach_volume(volume, worker_nodes)
+
+@libtest
+@ibmcloud_platform_required
+def test_detachvolume():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+
+    volume = ibmcloud.getVolumeId()
+
+    worker_nodes = node.get_nodes(node_type="worker", num_of_nodes=1)
+    ibmcloud.detach_volume(volume, worker_nodes)
+
+@libtest
+@ibmcloud_platform_required
+def test_get_node_by_attached_volume():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+
+    volume = ibmcloud.getVolumeId()
```

Comment:
The same fixture for creating volume which will return it's ID can be used here.